packages/dds/tree/src/feature-libraries/editable-tree/editableTreeContext.ts

```diff
@@ -53,10 +59,24 @@ export interface EditableTreeContext {
 }
 
 export class ProxyContext implements EditableTreeContext {
-    public readonly withCursors: Set<ProxyTarget> = new Set();
-    public readonly withAnchors: Set<ProxyTarget> = new Set();
+    public readonly withCursors: Set<BaseProxyTarget> = new Set();
+    public readonly withAnchors: Set<BaseProxyTarget> = new Set();
+    /**
+     * A reference to `NeverAnchor` of `AnchorSet`.
+     * Used in `BaseProxyTarget` to indicate that the target is empty.
+     */
+    public readonly neverAnchor: Anchor;
+    /**
+     * A cursor which is freed.
+     * Used as a placeholder in `BaseProxyTarget` for empty targets.
+     */
+    public readonly neverCursor: ITreeSubscriptionCursor;
 
-    constructor(public readonly forest: IEditableForest) {}
+    constructor(public readonly forest: IEditableForest) {
+        this.neverAnchor = forest.anchors.track(null);
+        this.neverCursor = forest.allocateCursor();
+        this.neverCursor.free();
```

Comment:
Totally unsure about this, but don't want to do it every time an empty target is created.

Please, note that the empty target is based only on a fact that its anchor is`NeverAnchor`.